app/presenters/licences/customer-contacts.presenter.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+
+/**
+ * Formats data for the `/licences/{id}/contact-details` view customer contact details page
+ * @module CustomerContactDetailsPresenter
+ */
+
+const ContactModel = require('../../models/contact.model.js')
+
+/**
+ * Formats data for the `/licences/{id}/contact-details` view customer contact details page
+ *
```

Comment:
NIT - Just the missing param

```suggestion
 * Formats data for the `/licences/{id}/contact-details` view customer contact details page
 *
 * @param {Object[]} customerContacts - The results from `FetchCustomerContactsService` to be formatted for the view
 * template
 *
 * @returns {Object} The data formatted for the view template
```